### DISCUSSION TITLE
How to prase the pdf and get it's element  to save in database

### DISCUSSION THREAD

**Discussion by 751320829** [jsvine/pdfplumber#889] [Ask for help with specific PDFs]
i have a amazon summary pdf
like this
![ILYld](https://github.com/jsvine/pdfplumber/assets/39520779/57b4a389-47a1-4ba3-851b-9d0fff57ddad)

i want to  convert to a object  like 

class a:
    Income=123.0

please tell me  how to do it

**Comment by jsvine** [comment]
If all you need is the income, you can do something like this:

```python
import pdfplumber
pdf = pdfplumber.open("path/to/my/file.pdf")
page = pdf.page[0]
result = page.search(r"Income\s+([\d,\.\-]+)")
```

... and then examine the `result` object.

**Reply by 751320829** [reply]
Does the pdfplumber can analysis the pdf with no border?

**Reply by jsvine** [reply]
Unfortunately, I don't think I understand this question. Could you explain it in more detail / with specific examples?
